src/main/java/com/adobe/cq/testing/junit/rules/ClassRuleUtils.java

```diff
@@ -31,7 +30,7 @@
     // TODO way to make these props homogenous?
     public static final String LOGIN_TOKEN_AUTH = "it.logintokenauth";
 
-    public static class LoginTokenInstance extends ExistingInstance {
+    public static class LoginTokenInstance extends ConfiguredIndexingLaneInstance {
```

Comment:
I don't think I like this. Can't we just have a separate `Rule` with a separate statement? `LoginTokenInstance` does not logically depend on indexing. We can link them together in the `CQAuthorRule`s top-level convenience rules.

Reply:
@dulvac, I'm sorry I'm not quite sure I understand your suggestion. Please consider me a complete novice here. Could you please guide me towards how exactly do you want this change to be implemented? (btw, as you might notice in earlier conversation - I don't want to invest time in refactoring other pieces of code for this change - that, imvho, should be taken up as a separate issue and done by someone well versed)

Reply:
@catholicon , no worries. What I mean by this is I think we have two options:

A: If it's a corner case and only affect certain tests
* `LoginTokenInstance` should NOT extend `ConfiguredIndexingLaneInstance` 
* `ConfiguredIndexingLaneInstance` should NOT extend `ExistingInstance`.
* Maybe we have a separate `ClassRule` which takes an `Instance` in the constructor, that you need to explicitly add to your test and call `setLaneNames` on the default client

B: if we want to enable it for _every_ test by default
* What @volteanu said, and I support it:
* Rename `LoginTokenInstance` to `ConfigurableInstance` and add your logic there.

Reply:
So, something like catholicon/aem-testing-clients@3bb4d16 maybe?